pages/results/round-one.html

```diff
@@ -0,0 +1,150 @@
+    </p>
+  </div>
+  <div class="tablet-lg:grid-col-6">
+    <blockquote class="smeqa-results-blockquote smeqa-results-blockquote--inline">
+      <p class="usa-intro smeqa-results-blockquote__text">
+        We got the results that you all promised. We have a list of 36 people who can do the job.
+      </p>
+      <p class="smeqa-results-blockquote__attribution">
+        &mdash; Hiring Manager at HHS
+      </p>
+    </blockquote>
+  </div>
+</div>
+
+<h3>Cost benefit analysis</h3>
+<p>
+  We averaged the time required of both HR specialists and SMEs from both pilot agencies that participated in the SME-QA process, which helped the team calculate the average cost for each selection made off of a certificate.
+</p>
+<p>
+  If the pilot agencies that used the SME-QA process made six selections off a certificate, the cost per hire would become equal to the industry average, and any selections over six would reduce the cost per hire considerably. If DOI and HHS had ended up making offers to all qualified applicants, their cost per hire would have dropped to between $700 - $1000.
+</p>
+<div class="grid-row grid-gap">
+  <div class="tablet-lg:grid-col-6">
+    <table class="usa-table">
+      <thead>
+        <tr>
+          <th scope="col">Role</th>
+          <th scope="col">FTA</th>
+          <th scope="col">Hours per FTE</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>SME</td>
+          <td>8</td>
+          <td>57.5</td>
+        </tr>
+        <tr>
+          <td>HR</td>
+          <td>2</td>
+          <td>115</td>
+        </tr>
+      </tbody>
+    </table>
+    <p class="smeqa-results-table__caption">Assumes 150-200 applicants</p>
+  </div>
+  <div class="tablet-lg:grid-col-6">
+    <table class="usa-table">
+      <thead>
+        <tr>
+          <th scope="col">Total hires</th>
+          <th scope="col">Cost per hire</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>5</td>
+          <td>$6,692</td>
+        </tr>
+        <tr>
+          <td>10</td>
+          <td>$3,346</td>
+        </tr>
+        <tr>
+          <td>15</td>
+          <td>$2,231</td>
+        </tr>
+        <tr>
+          <td>20</td>
+          <td>$1,673</td>
+        </tr>
+        <tr>
+          <td>25</td>
+          <td>$1,338</td>
+        </tr>
+        <tr>
+          <td>30</td>
+          <td>$1,115</td>
+        </tr>
+        <tr>
+          <td>Society for HR Management</td>
+          <td>$4,129</td>
```

Comment:
This number should be $4,425 if we want to update it for the average from their 2017 report.

I'm worried this metric isn't quite apple-to-apples for us. According to [the report](https://www.shrm.org/ResourcesAndTools/business-solutions/Documents/Talent-Acquisition-Report-All-Industries-All-FTEs.pdf): 

> Cost-per-hire represents the costs involved with a new hire. These costs include the sum of third-party agency fees, advertising agency fees, job fairs, online job board fees, employee referrals, travel cost of applicants and staff, relocation costs, recruiter pay and benefits, and talent acquisition system costs, divided by the number of hires.

Our calculation includes the HR hours, but not recruiting costs (though these are minimal) or the TAS costs.